core/organization/service.go

```diff
@@ -288,6 +290,8 @@ func (s Service) RemoveUsers(ctx context.Context, orgID string, userIDs []string
 		}); err != nil {
 			err = errors.Join(err, currentErr)
 		}
+
+		audit.GetAuditor(ctx, orgID).Log(audit.OrgMemberDeletedEvent, audit.UserTarget(userID))
```

Comment:
there is already a similar log in API package, should we remove that if we want to add it here?